src/spring/HISTORY.md

```diff
@@ -1,5 +1,9 @@
 Release History
 ===============
+1.24.5
```

Comment:
Adding new arg is taken as new feature, which according to cli extension schema, should increase minor version number to be 1.25.0. 

For more versioning info, please refer to this doc: https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md